docs/Release-Process.md

```diff
@@ -28,41 +28,23 @@
   $ ./update-repository-version.sh -p "$NEW_VERSION" "$BRANCH"
   ```
 
-### Merge all bump version Pull requests
+### Merge the bump version Pull request
 
-  - The above step will create a GitHub pull request in the Kata projects. Trigger the CI using `/test` command on each bump Pull request.
+  - The above step will create a GitHub pull request in the Kata repository. Trigger the CI using `/test` command on the bump Pull request.
   - Check any failures and fix if needed.
-  - Work with the Kata approvers to verify that the CI works and the pull requests are merged.
+  - Work with the Kata approvers to verify that the CI works and the pull request is merged.
 
-### Tag all Kata repositories
+### Tag the Kata repository
 
-  Once all the pull requests to bump versions in all Kata repositories are merged,
-  tag all the repositories as shown below.
+  Once all the pull request to bump version in the Kata repository is merged,
```

Comment:
Maybe drop the "all" here, since there is a single pull request.

Reply:
Leftover... Thanks ! Fixed.